packages/blocks/src/store/selectors.js

```diff
@@ -95,6 +95,44 @@ export function getBlockVariations( state, blockName, scope ) {
 	} );
 }
 
+/**
+ * Returns the active block variation for a given block based on its attributes.
+ *
+ * @param {Object}                state      Data state.
+ * @param {string}                blockName  Name of block (example: “core/columns”).
+ * @param {Object}                attributes Block attributes used to determine active variation.
+ * @param {WPBlockVariationScope} [scope]    Block variation scope name.
+ *
+ * @return {(WPBlockVariation|void)} Active block variation.
+ */
+// eslint-disable-next-line camelcase
```

Comment:
It all depends if we think that active block variation is clear enough for developers using this API. 😄

I assume that this API is here to stay so if we want to keep it experimental it should be:

`__experimentalGetActiveBlockVariation`

This way it will be omitted in documentation and ESLint won't complain.

Reply:
I'll mark this as resolved. Renamed it to be stable.